src/aig-common/old-common/old-common.module.ts

```diff
@@ -1,10 +1,14 @@
 import { NgModule } from '@angular/core';
```

Comment:
not use this module. restore this file as original.

Reply:
restore this code: [link](https://raw.githubusercontent.com/EneaSys/aig_console/develop/src/aig-common/old-common/old-common.module.ts)